scripting/pugsetup.sp

```diff
@@ -1918,6 +1936,27 @@ stock void EndMatch(bool execConfigs = true, bool doRestart = true) {
   }
 }
 
+public void GetMapVotePool() {
```

Comment:
Bit of a nitpick: I'd rename this to SetupMapVotePool since it doesn't return anything, just updates a global.